ChineseFilter/sections/whitelist.txt

```diff
@@ -1,6 +1,8 @@
 !
 ! White list. Fixing filtration errors (false-positive)
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/88741
+@@||sc02.alicdn.com^$domain=mp4er.com
```

Comment:
It's better to remove mp4er.com from this rule
`||alicdn.com^$domain=dyhs6.com|mp4er.com`